c++/src/capnp/compat/json.c++

```diff
@@ -212,7 +222,7 @@ kj::String JsonCodec::encodeRaw(JsonValue::Reader value) const {
 }
 
 void JsonCodec::decodeRaw(kj::ArrayPtr<const char> input, JsonValue::Builder output) const {
-  KJ_FAIL_ASSERT("JSON decode not implement yet. :(");
+  _::parseJsonValue(input, output);
```

Comment:
Why the indirection to a separate function? Why not move the whole definition of `JsonCodec::decodeRaw()` down to where `parseJsonValue()` is defined?